### ISSUE TITLE
Task Completed accessibility signal does not play if there are warnings

### ISSUE THREAD

**Issue by CodyJung** [microsoft/vscode#244258]
Does this issue occur when all extensions are disabled?: **Yes**

- VS Code Version: 1.98.2
- OS Version: Windows 11, v10.0.22631

Steps to Reproduce:

1. Enable accessibility.signals.taskCompleted
2. Run a task that only has info or warning problems
3. No sound plays upon completion

In `taskTerminalStatus.ts:97` you can see that we play the success audio cue if the exit code was 0 AND the number of problemMatcher matches was 0. We'll play the failure audio cue if the exit code was not 0 or there was an error.

**But** if the task exited with code 0 and warnings or info markers, it will fall into one of the final two else-ifs and no cue will play.

https://github.com/microsoft/vscode/blob/a4089e4ff0518295298917db2a79fbc21136511b/src/vs/workbench/contrib/tasks/browser/taskTerminalStatus.ts#L97
